.gitignore

```diff
@@ -30,13 +30,6 @@ Example/package-lock.json
 Example/viewgenerator.cache
 ViewPacker/tools/node_modules/
 !ReactViewResources/AMDLoader/AMDLoaderExports.js
-Example/.wpcache/
-Example/global.d.ts
-Example.Avalonia/.wpcache/
-Example.Avalonia/Generated/
-Example.Avalonia/node_modules/
-Example.Avalonia/viewgenerator.cache
-Example.Avalonia/global.d.ts
 Tests.ReactView/.wpcache/
```

Comment:
would you mind to perform a cleanup of this file and remove all ReactView stuff?

Reply:
done